docs/integration/infra.md

```diff
@@ -0,0 +1,276 @@
+title: Infrastructure Monitoring Integration
+description: Infrastructure performance monitoring - metrics, events, processes, packages, log collection and parsing for bare-metal servers, virtual machines and containers with support for Docker, containerd, Kubernetes, Swarm, Nomad, AWS ECS, AWS EKS, AKS, GKE.
+
+The Sematext Infra Integration uses [Sematext Agent](../agents/sematext-agent) to collect metrics, events, processes, and package info for bare-metal servers, virtual machines and containers. Container monitoring is supported for Docker, containerd, k8s, Swarm, Nomad, AWS ECS, AWS EKS, AKS, GKE and more.
+
+Sematext saves you time and effort by giving you a strong starting point for monitoring your infrastrcture. You will **not** have to:
+
+- figure out which metrics to collect and which ones to ignore
+- give metrics meaningful labels
+- hunt for metric descriptions in the docs so that you know what each of them actually shows
+- build charts to group metrics that you really want on the same charts, not N separate charts
+- figure out the most useful aggregation to use for each metric (min? max? avg? something else?)
+- build dashboards to combine charts with metrics you typically want to see together
+- set up basic alert rules
+
+Sematext provides all that and more out of the box.
+
+
+## Monitoring with Sematext Agent
+
+Sematext Agent collects various metrics and other info about servers, virtual machines, container hosts and containers and ships that to Sematext Cloud.
+
+First create an [Infra Monitoring App in Sematext Cloud](https://apps.sematext.com/ui/monitoring-create). You will be presented with installation instructions for many possible environments.
+
+
+## How it works
+
+A Golang-based [Sematext Agent](../agents/sematext-agent) will be installed on the host you want to monitor. This agent sends all infrastructure data to an Infra App whose token is configured during the Agent's installation.
+
+It is capable of running in two kinds of environments:
+- bare-metal, virtual machine - we provide rpm/deb packages for most popular linux distributions
+- containers - a special sematext/agent container image provides support for monitoring in container environments
+
+In both cases, installing the agent will start gathering all infrastructure data right away. Additionally, it will enable [Service Autodiscovery](../monitoring/autodiscovery) and [Logs Autodiscovery](../logs/discovery/intro/) features which make it possible to automatically start monitoring any of supported [Sematext integrations](./index) or to ship logs produced by any process or container.
```

Comment:
```suggestion
In both cases, installing the agent will start gathering all infrastructure data right away. Additionally, it will enable [Service Autodiscovery](../monitoring/autodiscovery) and [Logs Autodiscovery](../logs/discovery/intro/) features which make it possible to automatically start monitoring any supported [Sematext integrations](./index) or to ship logs produced by any process or container.
```